src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,13 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Check if the command mutates data in the address book
+     *
+     * @return true if the command mutates data in the address book
```

Comment:
@return … line can be omitted for this method as it says the same thing as the first sentence.